third_party/closure-compiler/runner/src/AmpCommandLineRunner.java

```diff
@@ -0,0 +1,34 @@
+import com.google.common.collect.ImmutableSet;
+import com.google.javascript.jscomp.CommandLineRunner;
+import com.google.javascript.jscomp.CompilerOptions;
+import com.google.javascript.jscomp.CustomPassExecutionTime;
+
+public class AmpCommandLineRunner extends CommandLineRunner {
+  
+  ImmutableSet<String> suffixTypes = ImmutableSet.of(
+      "log.dev.fine",
+      "log.dev.assert"
+      );
+
+  protected AmpCommandLineRunner(String[] args) {
+    super(args);
+  }
+
+  @Override
+  protected CompilerOptions createOptions() {
+    CompilerOptions options = super.createOptions();
+    AmpPass ampPass = new AmpPass(getCompiler(), suffixTypes);
+    options.addCustomPass(CustomPassExecutionTime.BEFORE_OPTIMIZATIONS, ampPass);
```

Comment:
@cramforce @erwinmombay I'm not a huge fun of something like this for main binary - I'd rather avoid an extra module loading dependency. We may decide that we no longer want to package `log.js` into every extension and instead use `AMP.dev` and `AMP.user` - but that should be an independent thing.